scripts/create_backup_pd

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+ZONE=us-central1-f
+BUCKET=jenkins-backup-image
+SHA=""
+TOOL_BUCKET=jenkins-backup-image
+
+while getopts z:b:s:t: arg; do
+  case ${arg} in
+    z) ZONE="${OPTARG}";;
+    b) BUCKET="${OPTARG}";;
+    s) SHA="${OPTARG}";;
+    t) TOOL_BUCKET="${OPTARG}";;
+    *) echo "Unrecognized argument ${OPTARG}"; exit 1;
+  esac
+done
+
+if ["$SHA" = ""];then
+  echo 'Missing SHA for secrets. Please use -s'
+  exit 1;
+fi
+
+sudo rm -rf /tmp/jenkins-backup-pd
```

Comment:
instead of removing at startup. You can create a cleanup method at exit

function cleanup {
sudo umount ..
rm -rf ${TMP_DIR} 
}

trap cleanup EXIT